AspNetCoreWeb/Infrastructure/SortOptionsProcessor{T,TEntity}.cs

```diff
@@ -46,12 +45,11 @@ private static IEnumerable<SortTerm> GetValidTerms(JqueryDataTablesParameters ta
                 yield break;
             }
 
-            var declaredTerms = GetTermsFromModel(typeof(T));
+            var declaredTerms = GetTermsFromModel(typeof(T)).ToList();
```

Comment:
Please remove `ToList()` we can add that if we are enumerating this more than once.

Reply:
Calling SingleOrDefault on line 52 does iterate the entire list right? Or am I mistaken?

Reply:
I'm sorry. You're right. I misunderstood.